### DISCUSSION TITLE
can i only print selected thread in ctx?

### DISCUSSION THREAD

**Discussion by acgmusic** [hugsy/gef#1130] [Q&A]
### Type of feature request

None

### Summary Description

gef config context.nb_lines_threads 1
if i set config as above, i found that gef only print the first thread in the threads list, but not currently selected one.

### Implementation idea/suggestion

maybe add a new configuration item, or just change the behavior when nb_lines_threads is set to 0?

**Comment by hugsy** [comment]
Not by default, but you can easily add a custom filter. See [the docs](https://hugsy.github.io/gef/commands/context/#adding-custom-context-panes) for more info on creating custom panes for `context`
